packages/libs/react-ui/src/components/Grid/Grid.css.ts

```diff
@@ -16,15 +16,12 @@ export const gridItemClass = style([
 ]);
 
 export const gapVariants = styleVariants({
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  '2xs': [sprinkles({ gridGap: '$2xs' })],
-  xs: [sprinkles({ gridGap: '$xs' })],
-  sm: [sprinkles({ gridGap: '$sm' })],
-  md: [sprinkles({ gridGap: '$md' })],
-  lg: [sprinkles({ gridGap: '$lg' })],
-  xl: [sprinkles({ gridGap: '$xl' })],
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  '2xl': [sprinkles({ gridGap: '$2xl' })],
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  '3xl': [sprinkles({ gridGap: '$3xl' })],
+  $2xs: [sprinkles({ gridGap: '$2xs' })],
+  $xs: [sprinkles({ gridGap: '$xs' })],
+  $sm: [sprinkles({ gridGap: '$sm' })],
+  $md: [sprinkles({ gridGap: '$md' })],
+  $lg: [sprinkles({ gridGap: '$lg' })],
+  $xl: [sprinkles({ gridGap: '$xl' })],
+  $2xl: [sprinkles({ gridGap: '$2xl' })],
+  $3xl: [sprinkles({ gridGap: '$3xl' })],
```

Comment:
These were intentionally left without the "$" prefix because they are not style values, but prop options. The idea was to keep the "$" only in the css as a way to use custom style values

Reply:
Trying to get rid of the ts-ignore-next line, but makes sense to revert for consistency